apps/test/unit/code-studio/pd/workshop_dashboard/EnrollmentsPanelTest.jsx

```diff
@@ -0,0 +1,191 @@
+import _ from 'lodash';
+import React from 'react';
+import {shallow, mount} from 'enzyme';
+import {assert} from 'chai';
+import sinon from 'sinon';
+import {Factory} from 'rosie';
+import EnrollmentsPanel from '@cdo/apps/code-studio/pd/workshop_dashboard/EnrollmentsPanel';
+import './workshopFactory';
+
+describe('EnrollmentsPanel', () => {
+  let server, loadEnrollments;
+
+  beforeEach(() => {
+    server = sinon.createFakeServer();
+    loadEnrollments = sinon.spy();
+  });
+
+  afterEach(() => {
+    server.restore();
+    server = null;
+  });
+
+  it('shows a spinner when enrollments are loading', () => {
+    const workshop = Factory.build('workshop');
+    const wrapper = shallow(
+      <EnrollmentsPanel
+        workshopId={String(workshop.id)}
+        workshop={workshop}
+        isLoadingEnrollments={true}
+        enrollments={[]}
+        loadEnrollments={loadEnrollments}
+      />
+    );
+    assert(wrapper.find('Spinner').exists(), 'Spinner was rendered');
+    assert(
+      !wrapper.find('WorkshopEnrollment').exists(),
+      'WorkshopEnrollment was not rendered'
+    );
+  });
+
+  it('shows WorkshopEnrollment when enrollments are done loading', () => {
+    const workshop = Factory.build('workshop');
+    const wrapper = shallow(
+      <EnrollmentsPanel
+        workshopId={String(workshop.id)}
+        workshop={workshop}
+        isLoadingEnrollments={false}
+        enrollments={[]}
+        loadEnrollments={loadEnrollments}
+      />
+    );
+    assert(
```

Comment:
Nit: To be symmetrical to the test above, you can assert that spinner is not rendered here.

Reply:
Great point!  I'll do that.